files/image_config/hostcfgd/hostcfgd

```diff
@@ -304,13 +304,33 @@ class HostConfigDaemon:
             add = False
 
         self.iptables.iptables_handler(key, data, add)
+        
+    def feature_status_handler(self, key, data):
+        status_data = self.config_db.get_table('FEATURE')
+        for key in status_data.keys():
+        if not key:
+            syslog.syslog(syslog.LOG_WARNING, "FEATURE key is missing")
+            return
+        status = status_data[key]['status']
+        if not status:
+            syslog.syslog(syslog.LOG_WARNING, "status is missing for {}".format(key))
+            return
+        if status == "enabled":
+            sudo systemctl enable key
+            sudo systemctl start key
+            syslog.syslog(syslog.LOG_INFO, "FEATURE {} is enabled and started".format(key))
+        if status == "disabled":
+            sudo systemctl stop key
+            sudo systemctl disable key
```

Comment:
You can't just call Linux commands in Python. You need to use a library call like 'popen'. You should write a wrapper function to run the commands. There are a number of examples throughout SONiC Python code (i.e., the `show` command). You can probably just copy/paste one with very minimal changes. Also, 'key' is a variable, and it will need to be formatted into the command.